File: test/fixtures/python/multiple_commands.py
import requests

response = requests.get('http://localhost:28139')


response = requests.get('http://localhost:28139/second')


with open('input_redirected_file.txt', 'rb') as f:
    data = f.read()

response = requests.put('http://localhost:28139/third', data=data)

File: test/fixtures/ruby-httparty/multiple_commands.rb
require 'httparty'

url = 'http://localhost:28139'
res = HTTParty.get(url)


url = 'http://localhost:28139/second'
res = HTTParty.get(url)


url = 'http://localhost:28139/third'
body = File.read('input_redirected_file.txt')
res = HTTParty.put(url, body: body)
